mlfinlab/feature_importance/importance.py

```diff
@@ -131,7 +143,7 @@ def mean_decrease_accuracy(model, X, y, cv_gen, sample_weight_train=None, sample
     if scoring == log_loss:
         importance = importance / -features_metrics_values
     else:
-        importance = importance / (1.0 - features_metrics_values)
+        importance = importance / (1.0 - features_metrics_values).replace(0, np.nan)
```

Comment:
Replace nan with 0 will be on line 148, so we can stay with original expression without replace

Reply:
I have done that because I get DivideByZero pandas error in the Travis CI build test.